qrcode/util.py

```diff
@@ -453,20 +450,17 @@ def write(self, buffer):
                 else:
                     buffer.put(ALPHA_NUM.find(chars), 6)
         else:
-            if six.PY3:
-                # Iterating a bytestring in Python 3 returns an integer,
-                # no need to ord().
-                data = self.data
-            else:
-                data = [ord(c) for c in self.data]
+            # Iterating a bytestring in Python 3 returns an integer,
+            # no need to ord().
+            data = self.data
             for c in data:
```

Comment:
I’d expect every Python developer to know that at this point, you can simplify into:
```suggestion
            for c in self.data:
```

Reply:
:)